website/routes.py

```diff
@@ -1124,6 +1142,31 @@ def make_url_map(app):
             '/project/<pid>/node/<nid>/get_registrations/',
         ], 'get', project_views.node.get_registrations, json_renderer),
 
+        # Draft Registrations
+        Rule([
+            '/project/<pid>/draft/',
+        ], 'get', project_views.drafts.get_draft_registrations, json_renderer),
+        Rule([
+            '/project/<pid>/draft/<draft_id>/',
+        ], 'get', project_views.drafts.get_draft_registration, json_renderer),
+        Rule([
+            '/project/<pid>/draft/',
+        ], 'post', project_views.drafts.create_draft_registration, json_renderer),
+        Rule([
+            '/project/<pid>/draft/<draft_id>/',
```

Comment:
A previous comment on URLs has not yet been addressed. Per Jeff in https://github.com/CenterForOpenScience/osf.io/pull/4338/#discussion-diff-40328074 :
> Why is it just draft? Probably want to be specific and have draft under the registration url.

Please make sure that all new URLs in `routes.py` pass URL review, and then post confirmation. Consults are Jeff and BrianG.